Edits to Git entries (11 total) (#562)
Codecademy/docs: content/git/concepts/init/init.md

> git init

Reviewer: ```suggestion
$ git init
```

```

## Initializing a Git Repository

> ### Step 1: Navigate to the Correct Directory

Reviewer: ```suggestion
### Step 1: Navigate to the Desired Directory
```

Reviewer: What do we think about replacing "appropriate directory" with "desired directory"?

Author: Agree, "desired" is better.

> The `init` command initializes a Git repository in the current directory. Prior to using this command, users must take care to navigate to the appropriate directory&mdash;typically, the root directory of the project. The `cd` command can be used to navigate the filesystem from the command line:

Reviewer: ```suggestion
The `init` command initializes a Git repository in the current directory. Prior to using this command, users must take care to navigate to the appropriate directory &mdash; this will become the root directory of the repository.

The `cd` command can be used to navigate the filesystem from the command line:
```

If using an em dash, let's surround them with a single space.

```shell

> cd go/to/desired/project/directory

Reviewer: ```suggestion
$ cd go/to/desired/project/directory
```

Author: Shouldn't this be a \`\`\`pseudo with no `$`?

Reviewer: I agree, yes, it should be. 😄

```

### Step 2: Init

From within the appropriate directory, running `git init` will initialize a repository and Git will begin tracking changes made to all the files within:

```shell

> git init

Reviewer: ```suggestion
$ git init
```
